chromium/browser_type.go

```diff
@@ -48,22 +48,16 @@ type BrowserType struct {
 // NewBrowserType registers our custom k6 metrics, creates method mappings on
 // the goja runtime, and returns a new Chrome browser type.
 func NewBrowserType(vu k6modules.VU) api.BrowserType {
-	var (
-		rt    = vu.Runtime()
-		hooks = common.NewHooks()
-	)
-
 	// NOTE: vu.InitEnv() *must* be called from the script init scope,
 	// otherwise it will return nil.
 	k6m := k6ext.RegisterCustomMetrics(vu.InitEnv().Registry)
 	b := BrowserType{
 		vu:        vu,
-		hooks:     hooks,
+		hooks:     common.NewHooks(),
 		k6Metrics: k6m,
 		storage:   &storage.Dir{},
 		randSrc:   rand.New(rand.NewSource(time.Now().UnixNano())), //nolint: gosec
 	}
-	rt.SetFieldNameMapper(common.NewFieldNameMapper())
```

Comment:
nit; from a very quick look it seems like you do not delete the FIeldNameMapper code inside the xk6-browser/common package, was there a reason to not delete it ?

Reply:
Ah, right. Thanks. Resolved in [21ad06b](https://github.com/grafana/xk6-browser/pull/673/commits/21ad06b0e0e7ac271c125a4759c9e31f971328c9).